chrometool/chrometool.go

```diff
@@ -3,62 +3,77 @@ package chrometool
 import (
 	"context"
 	"fmt"
+	"runtime"
+	"strings"
 
 	"github.com/chromedp/cdproto/cdp"
 	"github.com/chromedp/cdproto/dom"
+	"github.com/chromedp/cdproto/input"
 	"github.com/chromedp/chromedp"
+	"github.com/chromedp/chromedp/kb"
+	"github.com/jezek/xgb/xproto"
 )
 
+// NthChildSel hold information about the target node
+// Sel refers to the CSS selector and
+// position refers to their position among element nodes matching the selector.
+type NthChildSel struct {
+	Selector interface{}
```

Comment:
The usage below seems to indicate that this should be a string?

Reply:
They used selector as type interface in chromedp, so I kept it as well

Reply:
You can pass string to interface, but not the other way around. If we know we are constructing selector from a string (which we would, since we'd read it from config), we should be passing a string.

Reply:
got it